Src/AutoMoqUnitTest/MockRelayTest.cs

```diff
@@ -125,6 +126,35 @@ public void CreateReturnsCorrectResultWhenContextReturnsNonMock()
             // Teardown
         }
 
+        public static IEnumerable<object[]> ValidNonMockSpecimens
+        {
+            get
+            {
+                yield return new object[] {new NoSpecimen()};
+                yield return new object[] {new OmitSpecimen()};
+                yield return new object[] {null};
+            }
+        }
+
+        [Theory]
+        [PropertyData("ValidNonMockSpecimens")]
```

Comment:
Could you please turn this into a `[ClassData]`, so that it's not stringly typed?